src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/casenotes/CaseNotesNomisApiService.kt

```diff
@@ -21,4 +23,12 @@ class CaseNotesNomisApiService(@Qualifier("nomisApiWebClient") private val webCl
       .uri("/casenotes/{caseNoteId}", caseNoteId)
       .retrieve()
       .awaitBody()
+
+  suspend fun updateCaseNote(caseNoteId: Long, nomisCaseNote: UpdateCaseNoteRequest) {
+    webClient.put()
+      .uri("/casenotes/{caseNoteId}", caseNoteId)
+      .bodyValue(nomisCaseNote)
+      .retrieve()
+      .awaitBodilessEntity()
+  }
```

Comment:
The casenotes sync now does something unusual - when a Nomis case note changes, it potentially updates a related case note in Nomis

Reply:
Will that then raise an xtag event which will then be ignored?